app/main.py

```diff
@@ -1,2 +1,55 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+
+    CAPACITY = 8
+    LOAD_FACTOR = 2 / 3
+
+    def __init__(self) -> None:
+        self.threshold = int(self.CAPACITY * self.LOAD_FACTOR)
+        self.table = [[] for _ in range(self.CAPACITY)]
+        self.length = 0
+
+    def resize_table(self) -> None:
+        table_copy = self.table.copy()
+        self.length = 0
+        self.CAPACITY *= 2
+        self.threshold = int(self.CAPACITY * self.LOAD_FACTOR)
+        self.table = [[] for _ in range(self.CAPACITY)]
+        for x in table_copy:
+            if x:
+                self.__setitem__(x[0], x[1])
```

Comment:
You can use the usual interface for assigning items here, no need to call the magic method directly.